src/main/java/junit/extensions/ActiveTestSuite.java

```diff
@@ -1,9 +1,6 @@
 package junit.extensions;
 
-import junit.framework.Test;
-import junit.framework.TestCase;
-import junit.framework.TestResult;
-import junit.framework.TestSuite;
+import junit.framework.*;
```

Comment:
I prefer we avoid using globs in imports. I wouldn't go so far as to suggest to remove existing globs, but let's not add more.

Reply:
Reverted in 7b53777.